CHANGES.rst

```diff
@@ -24,6 +24,11 @@ jplsbdb
 - Fix a bug for jplsdbd query when the returned physical quantity contains
   a unit with exponential. [#2377]
 
+linelists/cdms
```

Comment:
I don't think we are checking for it, but the syntax here follows the namespace rather than the directory structure (separated with `.`)

Reply:
thanks, I didn't realize

Reply:
I think this is something we blindly inherited from core astropy, but I don't think we actually check for it anywhere. So at this point, it's just "historical reasons" 🤷‍♀️